backend/editor/entries.py

```diff
@@ -173,13 +177,28 @@ def update_nodes(label, entry, new_node_keys):
             continue
         query.append(f"""\nREMOVE n.{key}\n""")
 
+    #Normalising Node to be Updated
+    normalised_new_node_keys = {}
+    for keys in new_node_keys.keys():
+        if keys.startswith("tags") and not keys.endswith("str"):
+            if "ids" not in keys:

```

Comment:
add the "_" to make the condition even more specific and explicit

```suggestion
        if keys.startswith("tags_") and not keys.endswith("_str"):
            if "_ids_" not in keys:
                # generate tags_ids from values
```

Reply:
makes sense